binance/client.py

```diff
@@ -506,7 +506,28 @@ def get_symbol_info(self, symbol) -> Optional[Dict]:
                 return item
 
         return None
+    
+    def get_latest_price(self,symbol) -> int:
```

Comment:
The return type here would be a float or str, not an int

Reply:
Changed to float